test/unit/zeromq-node-gyp/output.js

```diff
@@ -1,9 +1,11 @@
 [
+  "node_modules/@aminya/node-gyp-build/index.js",
+  "node_modules/@aminya/node-gyp-build/package.json",
   "node_modules/zeromq/lib/draft.js",
   "node_modules/zeromq/lib/index.js",
   "node_modules/zeromq/lib/native.js",
   "node_modules/zeromq/lib/util.js",
   "node_modules/zeromq/package.json",
-  "node_modules/zeromq/prebuilds/darwin-x64/node.napi.glibc.node",
```

Comment:
@styfle  I thought during the tests the prebuilds would get picked up but they don't. I have seen the PR you gave me as example the output list also did not contain the binary: https://github.com/vercel/nft/pull/286/files#diff-fd366a1b2d1df15ab6dcbf1c7ee76e755a9ce56ee74e247e5657738061e65c9e (test/unit/microtime-node-gyp/output.js), so I am assuming it's OK, since if I install the package locally on a Next.js app it works fine.